ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-commands.sh

```
@@ -144,6 +144,10 @@ case "${PLATFORM}" in
       ARGS+=( --feature-set "${GUEST_FEATURE_SET}")
     fi
 
+    if [[ -n "${EXTRA_ARGS}" ]]; then
+      ARGS+=( ${EXTRA_ARGS} )
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)